.gitignore

```diff
@@ -8,6 +8,7 @@
 /stamp-h1
 .dep
 Makefile.in
+build
```

Comment:
It is up to the user to choose a name for the build folder, so this isn't really a general rule and we should probably not add it.

Reply:
Sounds good.